### DISCUSSION TITLE
How can I add a prefix for hasura in the reverse proxy ?

### DISCUSSION THREAD

**Discussion by Barolina** [hasura/graphql-engine#6820] [Question] [answered]
I have  a  domain   https://my-site.ru/
I want to  open the hasura   in  https://my-site.ru/hasura/   and  to see
  
https://my-site.ru/hasura/console and   other   api. 
But now  i have 404.

**Comment by ecthiender** [comment, marked as answer]
@Barolina what reverse proxy are you using? Can you share your config?

Btw, here are [some examples from the docs](https://hasura.io/docs/latest/graphql/core/deployment/enable-https.html#sample-configurations). Hope that helps.

**Reply by Barolina** [reply]
I use to nginx  and my a view  the  config  like  this:

```

location ~ /hasura {
            proxy_pass         http://localhost:8080;
            proxy_http_version 1.1;
            proxy_set_header Upgrade $http_upgrade;
            proxy_set_header Connection "upgrade";
        }
  }
```

And i open  http://192.168.2.2:8081/hasura in my browse  i have  like this a response:

```
{"path":"$","error":"resource does not exist","code":"not-found"}
```

**Reply by Barolina** [reply]
@ecthiender ,  Hi)) Do you remember about me , (my situation) ) ?

**Reply by Barolina** [reply]
Sorry and  thanks  you,  I didn't read the documentation carefully, I missed it with a slash at the end

**Reply by Barolina** [reply]
it is work ) 

```
  location ~ /hasura/ {
            proxy_pass         http://localhost:8080/;
            proxy_http_version 1.1;
            proxy_set_header Upgrade $http_upgrade;
            proxy_set_header Connection "upgrade";
        }
  }```
